runtime/src/main/scala/org/apache/pekko/grpc/javadsl/package.scala

```diff
@@ -40,20 +40,20 @@ package object javadsl {
     }
 
   /**
-   * Helper for creating Scala partial functions from [[pekko.japi.Function]]
-   * instances as Scala 2.11 does not know about SAMs.
+   * Helper for creating Scala partial functions from [[pekko.japi.function.Function]]
+   * instances.
    */
   @deprecated("no longer needed since support for Scala 2.11 has been dropped", "1.2.0")
```

Comment:
Should we remove this too

Reply:
I'll have a look at this again. I think it is currently needed.